funcs.el

```diff
@@ -10,21 +10,9 @@
 ;;; License: GPLv3
 
 ;;; Code:
-;;;; jupter repl minor mode keymap
-(spacemacs/set-leader-keys-for-minor-mode 'use-jupyter-repl-mode
-  "'"  'jupyter-run-repl
-  "cc" 'jupyter-load-file
-  "cC" '(lambda (file) (interactive "f\file: ") (progn (jupyter-load-file file) (jupyter-repl-pop-to-buffer)))
-  "sB" '(lambda () (interactive) (progn (jupyter-eval-buffer (current-buffer)) (jupyter-repl-pop-to-buffer)))
-  "sb" 'jupyter-eval-buffer
-  "sF" '(lambda () (interactive) (progn (jupyter-eval-defun) (jupyter-repl-pop-to-buffer)))
-  "sf" 'jupyter-eval-defun
-  "si" 'jupyter-run-repl
-  "sR" '(lambda () (interactive) (progn (jupyter-eval-line-or-region 'null) (jupyter-repl-pop-to-buffer)))
-  "sr" 'jupyter-eval-line-or-region)
 
 ;;;; jupyter repl minor mode
-(defvar use-jupyter-repl-mode nil)
+;; (defvar use-jupyter-repl-mode nil)
 
 (define-minor-mode use-jupyter-repl-mode
```

Comment:
I think this should be a local package, i.e. a file and then loaded in packages.el, this might also fix needing to use the keybinging twice

Reply:
The file should be in a subfolder local (see for example the +completion/helm layer with helm-spacemacs-help